src/components/CommandBar.js

```diff
@@ -53,18 +53,26 @@ const CommandBar = React.createClass({
 
   componentWillUnmount() {
     const shortcuts = this.context.shortcuts;
-    shortcuts.off("F8", this.props.resume);
-    shortcuts.off("F10", this.props.stepOver);
-    shortcuts.off(`${ctrlKey}F11`, this.props.stepIn);
-    shortcuts.off(`${ctrlKey}Shift+F11`, this.props.stepOut);
+    shortcuts.off("F8");
+    shortcuts.off("F10");
+    shortcuts.off(`${ctrlKey}F11`);
+    shortcuts.off(`${ctrlKey}Shift+F11`);
   },
 
   componentDidMount() {
     const shortcuts = this.context.shortcuts;
-    shortcuts.on("F8", this.props.resume);
-    shortcuts.on("F10", this.props.stepOver);
-    shortcuts.on(`${ctrlKey}F11`, this.props.stepIn);
-    shortcuts.on(`${ctrlKey}Shift+F11`, this.props.stepOut);
+    const handleEvent = (e, func) => {
+      e.preventDefault();
+      if (e.stopPropogation) {
```

Comment:
strange, i've never seen this `stopPropogation` not present. What case are you handling here?